src/snowflake/snowpark/_internal/code_generation.py

```diff
@@ -25,7 +25,7 @@
 GLOBAL_OPS = (STORE_GLOBAL, DELETE_GLOBAL, LOAD_GLOBAL)
 
 CODE_AS_COMMENT_HINT = (
-    "The following comment contains the UDF source code"
+    "The following comment contains the UDF/SP source code"
```

Comment:
How about removing `UDF`, so that it also applies to other UD(X)Fs? It is already inside UDF or SP so it should be self-evident.

Reply:
Is there any other UD(X)F than UDTF 🤔 ?

Reply:
UDAF :p